packages/react-api/package.json

```diff
@@ -53,11 +53,14 @@
     "webpack-cli": "^4.5.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.13.9"
+    "@babel/runtime": "^7.13.9",
+    "@turf/bbox-polygon": "^6.3.0",
```

Comment:
@padawannn I thought these turf deps were not required now here, are they?

Reply:
you're right